### PR TITLE
[ESPNET1] Add the results of conformer-transducer for Librispeech

### PR DESCRIPTION
**Pull request by eesungkim**

### CODE REVIEW

File: egs/librispeech/asr1/RESULTS.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+- Environments
+  - python version: `3.8.3 (default)  [GCC 7.3.0]`
+  - espnet version: `espnet 0.10.7a1`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.10.0`
+
+- Model files (archived to model.tar.gz by `$ pack_model.sh`)
+    - model link: ([pretrained model](https://drive.google.com/file/d/1LiJ8VYNHWyYocapBbjnQsKy5GuBLRSP-/view?usp=sharing))
+    - training config file: `conf/tuning/transducer/train_conformer-rnn_transducer.yaml`
+    - decoding config file: `conf/decode.yaml`
```
Comment (b-flo) [inline egs/librispeech/asr1/RESULTS.md:12]: Is it intended? Default decode config for LibriSpeech is linked to [decode_pytorch_transformer_large.yaml](https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/conf/tuning/decode_pytorch_transformer_large.yaml) which is used for CTC/Att model.

It's not an issue because only `beam-size` is taken into account and default arguments will be used, but the beam size should be decreased here (60 -> 10)

**Comment by eesungkim** [comment]
Hi @b-flo,

Thank you for your effort on this open source project.

I wanted to reproduce the result of conformer-transducer model without auxiliary losses as the purpose of the research baseline. I thought some of Espnet users wanted to have this like me. 

Sure, I will train new model with auxiliary CTC and auxiliary LM loss and share the results soon.

**Comment by b-flo** [comment]
Could you add the training config and decoding config (see previous review's comment), please? When it's done, I'll merge this PR. You can a new one later for the model trained w/ auxiliary task!

File: egs/librispeech/asr1/RESULTS.md (hunk 1 above)
Comment (eesungkim) [reply egs/librispeech/asr1/RESULTS.md:12]: Surprisingly, the result with beam size 10/60 are not different on this model.

File: egs/librispeech/asr1/RESULTS.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
+# pytorch large conformer-transducer with specaug + speed perturbation (4 GPUs)
+
+- Environments
+  - python version: `3.8.3 (default)  [GCC 7.3.0]`
+  - espnet version: `espnet 0.10.7a1`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.10.0`
+
+- Model files (archived to model.tar.gz by `$ pack_model.sh`)
+    - model link: ([pretrained model](https://drive.google.com/file/d/1LiJ8VYNHWyYocapBbjnQsKy5GuBLRSP-/view?usp=sharing))
```
Comment (b-flo) [inline egs/librispeech/asr1/RESULTS.md:10]:
```suggestion
    - model link: ([pretrained model](https://drive.google.com/file/d/1fdadICi2w_b6lqb9_7J3wfRJc3LTnnSq/view?usp=sharing))
```

We have a gdrive for all ESPnet1 models, I uploaded your model on it.

**Comment by b-flo** [comment]
> The model w/ auxilary task is on training. When it's done, I'm going to add the results and the model including training/decoding config!

I think there is some confusion, sorry.
I meant to say you forgot to include the training and decoding configs for the model without auxiliary tasks in `egs/librispeech/asr1/conf/tuning/transducer`. You only updated the `RESULTS.md`.

File: egs/librispeech/asr1/RESULTS.md (hunk 1 above)
Comment (b-flo) [reply egs/librispeech/asr1/RESULTS.md:12]:
```suggestion
    - decoding config file: `conf/tuning/transducer/decode.yaml`
```

File: egs/librispeech/asr1/conf/tuning/transducer/decode.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+batchsize: 0
+beam-size: 60
+ctc-weight: 0
+penalty: 0.0
+search-type: maes #"nsc", "tsd", "alsd", "maes"
```
Comment (b-flo) [inline egs/librispeech/asr1/conf/tuning/transducer/decode.yaml:5]: See this [comment](https://github.com/espnet/espnet/pull/4080#discussion_r809960203). Your previous decoding config is equivalent to the one I shared (with `beamsize` decreased).

If you want to use mAES instead, you will need to tune the algorithm parameters accordingly (see the [documentation](https://espnet.github.io/espnet/tutorial.html#inference)).

File: egs/librispeech/asr1/conf/tuning/transducer/train_conformer-rnn_transducer.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+dropout-rate-decoder: 0.1
+## joint network related
+joint-dim: 512
+
+# transducer related
+model-module: "espnet.nets.pytorch_backend.e2e_asr_transducer:E2E"
+
+# reporter related
+report-wer: True
+report-cer: True
```
Comment (b-flo) [inline egs/librispeech/asr1/conf/tuning/transducer/train_conformer-rnn_transducer.yaml:53]: You can remove these lines, we disable CER/WER reporting for end-users (less training time).

File: egs/librispeech/asr1/RESULTS.md [hunk 5]
```diff
@@ -1,11 +1,42 @@
...
+- Environments
+  - python version: `3.8.3 (default)  [GCC 7.3.0]`
+  - espnet version: `espnet 0.10.7a1`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.10.0`
+
+- Model files (archived to model.tar.gz by `$ pack_model.sh`)
+    - model link: ([pretrained model](https://drive.google.com/file/d/1fdadICi2w_b6lqb9_7J3wfRJc3LTnnSq/view?usp=sharing))
+    - training config file: `conf/tuning/transducer/train_conformer-rnn_transducer.yaml`
+    - decoding config file: `conf/decode.yaml`
```
Comment (b-flo) [inline egs/librispeech/asr1/RESULTS.md:12]:
```suggestion
    - decoding config file: `conf/tuning/transducer/decode.yaml`
```

File: egs/librispeech/asr1/RESULTS.md [hunk 6]
```diff
@@ -1,11 +1,42 @@
...
+```
 # pytorch large conformer with specaug + speed perturbation (8 GPUs) + Transformer LM (4 GPUs)
 
 We used the same LM model in the previous report.
 
   - Model files (archived to model.tar.gz by `$ pack_model.sh`)
     - model link: ([pretrained model](https://drive.google.com/drive/folders/1GFaqEKsI6M_dBbxHD7s84J3yi5WGeNNC?usp=sharing))
     - training config file: `conf/tuning/conformer/train_pytorch_conformer_large.yaml`
-    - decoding config file: `conf/decode.yaml`
+    - decoding config file: `conf/tuning/transducer/decode.yaml`
```
Comment (b-flo) [inline egs/librispeech/asr1/RESULTS.md:39]:
```suggestion
    - decoding config file: `conf/decode.yaml`
```

**Comment by b-flo** [comment]
@eesungkim Sorry, I thought my manual changes would be the last ones but there is another one: could you please move the results section for transducer just before: 

> ## Lightweight/Dynamic convolution results

It's better to keep the best models at the top of the file for end users.

I'll merge this PR after that.

File: egs/librispeech/asr1/conf/tuning/transducer/decode.yaml (hunk 3 above)
Comment (eesungkim) [reply egs/librispeech/asr1/conf/tuning/transducer/decode.yaml]: Okay. Thanks!

**Comment by b-flo** [comment]
Merged, thanks again for your work @eesungkim !
